Formula/clblast.rb

```diff
@@ -0,0 +1,16 @@
+class Clblast < Formula
+  desc "Tuned OpenCL BLAS library"
+  homepage "https://github.com/CNugteren/CLBlast"
+  url "https://github.com/CNugteren/CLBlast/archive/1.3.0.tar.gz"
+  sha256 "fd5e06c261551366b46ccbc7841012cc69b58c74ed641db4f7a157c055d24223"
+  depends_on "cmake" => :build
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    pkgshare.install "samples/sgemm.c" # for a simple library linking test
+  end
+  test do
+    system ENV.cc, pkgshare/"sgemm.c", "-I#{include}", "-L#{lib}",
+                   "-lclblast", "-framework", "OpenCL"
```

Comment:
Could we run the test, and check its output?

Reply:
This is a sample program, so the 'test' here is to see if it compiles and links. CLBlast has a lot of real correctness tests, but they need to be explicitly compiled and run on an OpenCL device. From a Formula of a very similar library (clBLAS), I saw that [OpenCL doesn't work](https://github.com/Homebrew/homebrew-core/blob/7087358a3e8938a500bd7f8c67967821b58666d6/Formula/clblas.rb#L30) on the test systems?